packages/web/components/Authorization/NotAuthorized.js

```diff
@@ -3,12 +3,12 @@ import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
 
-const NotAuthorized = ({ size }) => {
+const NotAuthorized = ({ size, messageContext }) => {
 	const { t } = useTranslation(['error']);
 
 	return (
 		<Container size={size} data-testid="notAuthorized">
-			<h1>{t('error:notAuthorized')}</h1>
+			<h1>{t('error:notAuthorized', { context: messageContext })}</h1>
```

Comment:
I just think that it would be nice if we could make this message look a bit better. Maybe with a smaller container around it, white background and an icon like an alert sign, just so it doesn't look so simple.

Reply:
I think it would be better too. Maybe we can add this to our refactor list

Reply:
In the future we can improve this component with suggestions like these. @JoaoPauloSMoura can help us with this 🙂